### ISSUE TITLE
Accessibility issue: Purpose of social media footer links unclear

### ISSUE THREAD

**Issue by galvarez421** [PurdueMarketingAndMedia/purdueTemplates-2015#18]
Following up on the change made to fix #9, using `aria-label` instead of `title` attribute seems to be better in this case given that the links have no text content and therefore the purpose of the links should be conveyed in some other way. While the fix to #9 fixes the issue of invalid HTML, it arguably leaves open the described accessibility issue. I referred to the following resources:

1. https://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms-refs - Guideline 2.4.4
2. https://www.w3.org/TR/wai-aria/roles#textalternativecomputation - Explanation of how text alternatives are computed: 

> Authors can use the current section as a guide for creating names and descriptions in their markup.

> The last resort is to use text from a tooltip attribute (such as the title attribute in HTML). This is used only if nothing else, including subtree content, has provided results.

3. https://www.w3.org/WAI/GL/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html - Understanding success criterion 2.4.4

4. https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H33 - Technique H33 for satisfying criterion 2.4.4

> Because of the extensive user agent limitations in supporting access to the title attribute, authors should use caution in applying this technique.

5. https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/ARIA8 - Technique ARIA8 for satisfying criterion 2.4.4

> The aria-label attribute provides a way to place a descriptive text label on an object, such as a link, when there are no elements visible on the page that describe the object.

Note that the recommended change clears 7 "errors" according to [WAVE web accessibility evaluation tool Chrome extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh), although I think the recommended change would be more of an improvement than a "fix".

**Comment by galvarez421** [comment]
Another related resource: http://fontawesome.io/accessibility/

> In the case of focusable interactive elements, there are various options to include an alternative text or label to the element, without the need for any visually hidden <span> or similar. For instance, simply adding the aria-label attribute with a text description to the interactive element itself will be sufficient to provide an accessible alternative name for the element.

**Comment by jacobvenable** [comment]
Addressed in #78
